content/docs/intro/concepts/state.md

```diff
@@ -13,124 +13,81 @@ menu:
 aliases: ["/docs/reference/state/"]
 ---
 
-## State
+Pulumi stores metadata about your infrastructure so that it can manage your cloud resources. This metadata is called _state_ and is how Pulumi knows when and how to create, read, delete, or update cloud resources. Pulumi stores this metadata, the state, in a _backend_ of your choosing. Options include the Pulumi Service, a secure and reliable hosted application with policies and safeguards to facilitate team collaboration, in addition to object storage in AWS, Azure, GCP, or a simple filesystem. Most of the time, you don't need to explicitly worry about state and backends, but in some cases you may.
```

Comment:
why do we not need to worry about state? This is a question I continually get asked

Reply:
Updated to:

> Most of the time, you don't need to explicitly worry about state and backends, as the default options are designed to work seamlessly out-of-the-box, but in some cases you may choose to more explicitly configure backend options.

A bit more color on my take (this was our mindset when initially designing the SaaS):

I think of this like AWS CloudFormation. Most users think that CloudFormation doesn't have "state." Of course it does! But why do folks think that that? Because CloudFormation (like ARM and other services) was designed to abstract this away from the end user experience. The state is held within a database, hidden from end users, within the managed CloudFormation service. We have attempted to offer the same experience with Pulumi's Service backend. I personally wouldn't want to use a system that made me think of state regularly, as it's just a messy detail that we all wish didn't need to exist. Many of our end users are coming from Terraform, where "it's all about the state," however, which I think leads them to go down these paths probably earlier than is strictly required. Plus, the simple reality is that there _is_ state and because we're not a managed AWS service, there are absolutely cases where state is an unavoidable reality.

Reply:
👌